src/go/rdctl/cmd/snapshotList.go

```diff
@@ -80,10 +91,10 @@ func tabularOutput(snapshots []snapshot.Snapshot) error {
 		return nil
 	}
 	writer := tabwriter.NewWriter(os.Stdout, 0, 4, 4, ' ', 0)
-	fmt.Fprintf(writer, "ID\tName\tCreated\n")
+	fmt.Fprintf(writer, "Name\tCreated\n")
```

Comment:
Table headers should be all uppercase by convention:

```suggestion
	fmt.Fprintf(writer, "NAME\tCREATED\n")
```